lib/stigtest.pm

```diff
@@ -0,0 +1,63 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Base module for STIG test cases
+# Maintainer: llzhao <llzhao@suse.com>
+
+package stigtest;
+
+use strict;
+use warnings;
+use testapi;
+use utils;
+use base 'opensusebasetest';
+
+our @EXPORT = qw(
+  $profile_ID
```

Comment:
@lilyeyes  would you please list the items base on the code sequence?

Reply:
I don't understand you question

Reply:
I mean the list sequence as following variables you defined in this pm file.

Reply:
Oh, got it.
I think it is not necessary or maybe not good.
I just list the parameters together which are similar, it is readable somehow. such as list them together: $f_stdout
  $f_stderr   $f_report